public/themes/g.css

```diff
@@ -513,3 +513,37 @@ div.id4 {
     background: hsl(39, 74%, 60%);
 }
 
+
+/** handle the notificaiton color and the text color based on the theme **/
+/** handle the notificaiton color and the text color based on the theme **/
+.Toastify {
+    --toastify-font-family: "Open Sans", arial, sans-serif;
+  }
+  
+  .Toastify__toast-body {
```

Comment:
Most of this css seems to be common across the 5 themes, so it should probably be in lrr.css for now.

Reply:
Sorry that I just struggled into themes and not even notices the lrr.css.